tasks/system_probe.py

```diff
@@ -610,23 +611,28 @@ def ninja_generate(
 
 
 @task
-def build_libpcap(ctx):
+def build_libpcap(ctx, version=LIBPCAP_VERSION, sha256sum=LIBPCAP_SHA256SUM, force=False):
```

Comment:
Is building `libpcap` required to build system-probe?

Reply:
It is since this PR #30523 was merged